### ISSUE TITLE
mfa thirdparty download urllib.error.HTTPError: HTTP Error 404: Not Found in 2.0.0.a19

### ISSUE THREAD

**Issue by nrgslp** [MontrealCorpusTools/Montreal-Forced-Aligner#304]
Hi Michael - I am having trouble getting the third party binaries installed on the Debian VM that I use as the submitting machine to our HTC cluster. I am inside of a Singularity container because of they way that sndfile needed to be available to the worker nodes - and I don't know enough to know if the Singularity container is complicating things here. 

I'm using 2.0.0.a19.

```
Singularity> mfa thirdparty validate
The folder /home/.../Documents/MFA/thirdparty/bin does not exist
Singularity> mfa thirdparty download
0.00B [00:00, ?B/s]
urllib.error.HTTPError: HTTP Error 404: Not Found
Singularity>
```

What should the target URL be for this command? Can you confirm that it points to the correct location?
 
Thanks!

**Comment by mmcauliffe** [comment]
If you update to the latest version, it should be fixed as of 2.0.0a21.

**Comment by nrgslp** [comment]
Awesome, thanks!
